### ISSUE TITLE
Impact Analysis Graph and enhancement

### ISSUE THREAD

**Issue by mohammedbahja** [policycompass/policycompass#536]
The impact analysis results needs to be visualized.

**Comment by mohammedbahja** [comment]
The graph dose not work on the stage. 
The error is in app/modules/visualization/js/objectLine.js
I can see new changes have been added 
```
if (eventsData.length > 0) {
self.margin.top = self.margin.top + (self.spaceBetweenEvents * (maxEventsByPeriod + 1));
}
```
This code is checking "eventsData.length > 0" while eventsData is null in FCM case.
Can I change line of code at 2899 from  `if (eventsData.length > 0) { ` to  `if (eventsData!=null && eventsData.length > 0) { `
The same changes for line 2884 should be done which handle the same exception

![1](https://cloud.githubusercontent.com/assets/16784021/14847761/53b7c088-0c62-11e6-8217-74a0916e55e3.png)



I have edited the code locally form the browser developer tool and it worked.
